recipe/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+if [[ $target_platform == linux-aarch64 ]]; then
+    cmake   ${CMAKE_ARGS} \
+            -DCMAKE_INSTALL_PREFIX=$PREFIX \
+            -DCMAKE_BUILD_TYPE=Release \
+            -DCMAKE_CXX_COMPILER=clang++ \
```

Comment:
why does this need to be specified on `linux-aarch64`? doesn't it use the same toolchain as other unix platforms?